Gemfile.lock

```diff
@@ -3,7 +3,7 @@ PATH
   specs:
     codeclimate (0.14.2)
       activesupport (~> 4.2, >= 4.2.1)
-      codeclimate-yaml (~> 0.5.0)
+      codeclimate-yaml (~> 0.6.0)
```

Comment:
Is this dependency true? The [gemspec still specifies 0.5](https://github.com/codeclimate/codeclimate/blob/master/codeclimate.gemspec#L21)